package.json

```diff
@@ -24,7 +24,7 @@
     "lerna": "^3.22.1",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.6.2",
-    "typescript": "~4.6.3"
+    "typescript": "^4.7.1-rc"
```

Comment:
I know this is still draft, but given that `typescript` doesn't follow semver, we need to use `~` to only pick up patch updates and explicitly bump "minor" which are roughly majors for TS.